merkle.h

```diff
@@ -0,0 +1,113 @@
+#include "winternitz.h"
```

Comment:
"winternitz"? :o

Reply:
@l4l The signatures still need a lot of storage and Winternitz involves iterating hashes. Sorry I forgot to push `winternitz.h`. I'll push it soon.

Reply:
ah, nope, nvm
Just never heard of Wintermitz OTSS

Reply:
@l4l Ohh...I see, I'll have the code ready and now push it here in this PR for review.